sweetest/sweetest-core/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
By adding `**/build` to the root `.gitignore` we don't need dedicated `.gitignore` files in every module.